ynh-dev

```diff
@@ -145,6 +147,21 @@ elif [ "$1" = "run" ]; then
 
 
 #####################
+## Halt running VM ##
+#####################
+elif [ "$1" = "halt" ]; then
+    vm=$(vagrant global-status | head -3 | tail -1)
+    vm_status=$(echo $vm | cut -d ' ' -f 4)
```

Comment:
on my host (debian jessie, Vagrant 1.6.5), `cut -d ' ' -f 4` does not return anything.

Here is what $vm contains : `07c029b  old-stable      virtualbox poweroff /path/to/yunohost/devenv/stable`; MAybe playing with this good old `awk` command would be more robust.